code/modules/admin/topic.dm

```diff
@@ -982,7 +982,9 @@
 
 	else if(href_list["removenote"])
 		var/note_id = href_list["removenote"]
-		remove_note(note_id)
+		if(note_id)
+			if((alert("Do you really want to delete this note?","Note deletion confirmation", "Yes", "No") == "Yes") && note_id)
+				remove_note(note_id)
```

Comment:
Ok you check the note ID on 985, then again on 986, then again inside the proc (sql_notes.dm line 95).


Clean this up.

Reply:
gotta make sure my note didn't go walkies.
(I'll change it)